cli/integration/tests/waiter/cli.py

```diff
@@ -244,9 +244,9 @@ def delete(waiter_url=None, token_name=None, flags=None, delete_flags=None):
     return cp
 
 
-def ping(waiter_url=None, token_name=None, flags=None, ping_flags=None):
+def ping(waiter_url=None, token_name_or_service_id=None, flags=None, ping_flags=None):
     """Pings a token via the CLI"""
-    args = f'ping {token_name} {ping_flags or ""}'
+    args = f'ping {token_name_or_service_id or ""} {ping_flags or ""}'
```

Comment:
Why should we allow for `token_name_or_service_id` to be empty? Do we have a test where it checks the error message when it is not provided?

Reply:
Yes, `test_ping_invalid_args` does just that